src/analyses/goto_rw.h

```diff
@@ -78,10 +78,17 @@ class range_spect
 
   static range_spect to_range_spect(const mp_integer &size)
   {
-    PRECONDITION(size.is_long());
+    if(!size.is_long())
```

Comment:
❓ When wouldn't it be a long? A comment would help.

Reply:
Comment added, including an example. (A signed long, for example, isn't sufficient to capture a maximum size_t value.)